modules/Bio/EnsEMBL/Analysis/Hive/Config/TranscriptSelection.pm

```diff
@@ -0,0 +1,910 @@
+      $self->o('selected_projection_db'),
+      $self->o('long_read_final_db'),
+    ],
+
+    utr_donor_dbs => [
+      $self->o('cdna_db'),
+      $self->o('rnaseq_for_layer_db'),
+      $self->o('long_read_final_db'),
+    ],
+
+    utr_acceptor_dbs => [
+      $self->o('layering_db'),
+    ],
+
+    'utr_biotype_priorities' => {
+      'rnaseq' => 2,
+      'cdna'   => 1,
+    },
+
+    'cleaning_blessed_biotypes' => {
+      'pseudogene'           => 1,
+      'processed_pseudogene' => 1,
+      'IG_C_gene'            => 1,
+      'IG_V_gene'            => 1,
+      'TR_C_gene'            => 1,
+      'TR_D_gene'            => 1,
+      'TR_V_gene'            => 1,
+      'lncRNA'               => 1,
+    },
+
+    ensembl_analysis_script => catdir( $self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts' ),
+    remove_duplicates_script_path => catfile( $self->o('ensembl_analysis_script'), 'find_and_remove_duplicates.pl' ),
+    remove_small_orf_script => catfile( $self->o('ensembl_analysis_script'), 'genebuild', 'remove_small_orf.pl' ),
+
+    # cutoffs for removing small_orf genes
+    'small_orf_cutoff' => '100',
+    'intron_cutoff'    => '75',
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+
+########################
+# Executable paths
+########################
+
+    'blast_type' => 'ncbi',    # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# No option below this mark should be modified
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+
+########################
+# db info
+########################
+
+    'cdna_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_cdna_' . $self->o('release_number'),
+      -host   => $self->o('cdna_db_server'),
+      -port   => $self->o('cdna_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genblast_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_genblast_' . $self->o('release_number'),
+      -host   => $self->o('genblast_db_server'),
+      -port   => $self->o('genblast_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    long_read_final_db => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_lrfinal_' . $self->o('release_number'),
+      -host   => $self->o('long_read_final_db_server'),
+      -port   => $self->o('long_read_final_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'selected_projection_db' => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_sel_proj_' . $self->o('release_number'),
+      -host   => $self->o('projection_db_server'),
+      -port   => $self->o('projection_db_port'),
```

Comment:
Could you please change projection_db to selected_projection_db? This is to make the config file more explicit. This is to match line 90 91